android-json-form-wizard/src/main/java/com/vijay/jsonwizard/filesource/DiskFileSource.java

```diff
@@ -31,7 +32,7 @@ public class DiskFileSource implements FormFileSource {
     private MVELRuleFactory mvelRuleFactory;
 
     private DiskFileSource() {
-        this.mvelRuleFactory = new MVELRuleFactory(new YamlRuleDefinitionReaderExt());
+            this.mvelRuleFactory = new MVELRuleFactory(new YamlRuleDefinitionReaderExt());
```

Comment:
Kindly make this optional by switching to `YamlRuleDefinitionReader` or `YamlRuleDefinitionReaderExt`